omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -24,6 +23,17 @@ Homebrew requires the latest version of Xcode. Install :program:`Xcode` and
 the Command Line Tools for Xcode from the App Store. If you have already
 installed it, make sure all the latest updates are installed.
 
+Homebrew
+^^^^^^^^
+
+.. _`Homebrew wiki`: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Installation.md
+
+Homebrew will install all packages under :file:`/usr/local`. See also: Installation instructions on the `Homebrew wiki`_.
```

Comment:
The Homebrew wiki link https://github.com/Homebrew/legacy-homebrew/blob/master/share/doc/homebrew/Installation.md points to an archived page, I think it is a good opportunity now to replace it with https://github.com/Homebrew/brew/blob/master/docs/Installation.md, which is where the archived page suggests to go.

Reply:
Well spotted. I'll update it just now.